org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CodeActionHandler.java

```diff
@@ -332,10 +332,14 @@ public static IProblemLocationCore[] getProblemLocationCores(ICompilationUnit un
 					arguments.add(e.getAsString());
 				}
 			}
+			if (diagnostic.getData() instanceof String[] data) {
+				for (String s : data) {
+					arguments.add(s);
+				}
+			}
 			locations[i] = new ProblemLocationCore(start, end - start, problemId, arguments.toArray(new String[0]), isError, IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER);
 		}
-		return locations;
-	}
+		return locations;	}
```

Comment:
I would just clean up the formatting here.

Reply:
Fixed.